packages/provider/src/language-model/v2/language-model-v2-call-options.ts

```diff
@@ -130,5 +130,5 @@ Only applicable for HTTP-based providers.
    * to the provider from the AI SDK and enable provider-specific
    * functionality that can be fully encapsulated in the provider.
    */
-  providerOptions?: LanguageModelV2ProviderOptions;
+  providerOptions?: ProviderV2Options;
```

Comment:
Please keep specs fully separate for now.

Reply:
I've extracted `SharedV2ProviderOptions` and `SharedV2ProviderMetadata` that we can use in all model specs: https://github.com/vercel/ai/pull/5733

Reply:
awesome, thanks - yeah think it makes sense